content/en/docs/concepts/instrumentation/_index.md

```diff
@@ -5,6 +5,8 @@ aliases: [instrumenting]
 weight: 15
 ---
 
+![ Native Instrumentation vs instrumentation libraries](/img/libraries-instrumentation.svg)
```

Comment:
update this with the right URL after you have moved the file to the right location